### PR TITLE
PEP 641: Using an underscore in version portion of Python 3.10 compatibility tags

### PR DESCRIPTION
**Pull request by brettcannon**

### CODE REVIEW

**Review by warsaw — APPROVED** [review]

File: pep-0641.rst [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+
+The ``SOABI`` configure variable and
+``sysconfig.get_config_var('py_version_nodot')`` will be updated to
+use ``3_10`` appropriately.
+
+
+Backwards Compatibility
+=======================
+
+Tools relying on the 'packaging' project [2]_ already expects a
```
Comment (hugovk) [inline pep-0641.rst:63]:
```suggestion
Tools relying on the 'packaging' project [2]_ already expect a
```

File: pep-0641.rst [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+version specification of ``3_10`` for Python 3.10. Keeping the version
+specifier as ``310`` would require backing that change out and
+updating dependent projects (e.g. pip).
+
+Switching to ``3_10`` will require any tools that implicitly rely on
+the convention that the minor version is a single digit are implicitly
+broken regardless of any change here.
```
Comment (hugovk) [inline pep-0641.rst:70]: This sentence feels incomplete. Seems to be a missing verb?

File: pep-0641.rst [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+As use of the interpreter tag is mostly machine-based and this PEP
+disambiguates, there should not be any special teaching consideration
+required.
+
+
+Reference Implementation
+========================
+
+A pull request [1]_ already exists adding support to CPython 3.10.
+Support for reading wheel files with this proposed PEP already
```
Comment (hugovk) [inline pep-0641.rst:98]:
```suggestion
Support for reading wheel files with this proposed PEP is already
```

File: pep-0641.rst [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+implemented.
+
+
+Rejected Ideas
+==============
+
+Not making the change
+---------------------
+It was considered to not change the tag and stay with ``310``. The
+argument was it's less work and it won't break any preexisting
```
Comment (hugovk) [inline pep-0641.rst:108]: Is "existing" enough here? 
```suggestion
argument was it's less work and it won't break any existing
```

**Review by hugovk — COMMENTED** [review]
A few minor suggestions

**Pull request merged**